torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -183,8 +183,14 @@ std::shared_ptr<FutureMessage> RequestCallbackImpl::processRpc(
       auto& ctx = RRefContext::getInstance();
       std::shared_ptr<OwnerRRef> rref = ctx.getOwnerRRef(prf.rrefId());
       if (rref->hasValue()) { // optional fast-path
-        SerializedPyObj result = PythonRpcHandler::getInstance().serialize(
-            jit::toPyObject(rref->getValue()));
+        auto value = rref->getValue();
+        py::object pyValue;
```

Comment:
Is creating `py::object` out of GIL is OK?

Reply:
It's ok.
`inc_ref()` is called in `py::object`'s copy constructor with a reference passed in.
Here, ssing default constructor is like `std::make_shared(nullptr)`.